client/components/searchLiveTiHistory.js

```diff
@@ -0,0 +1,41 @@
+const moment = require('moment');
+
+const formatLiveTiRow = row => {
+  const unwrappedValues = row.map(cell => cell.formattedValue);
+  const time = moment(unwrappedValues[0], 'MM/DD/YYYY HH:mm:ss').format('MMMM Do YYYY');
+  const status = unwrappedValues[5];
+  const prompt = unwrappedValues[10];
+  return [time, status, prompt].filter(el => el).join(', ');
```

Comment:
Interesting filter.

Reply:
It gets rid of falsy elements, which in this case avoids returning strings like "July 28th 2018, No Show, ".